src/widgets/AutocompleteSelector.js

```diff
@@ -95,11 +96,10 @@ export class AutocompleteSelector extends React.PureComponent {
           autoCapitalize="none"
           autoCorrect={false}
           autoFocus
-          color="white"
-          onChange={text => this.setState({ queryText: text })}
           placeholder={placeholderText}
-          placeholderTextColor="white"
+          onChangeText={this.onChangeText}
           style={[localStyles.text, localStyles.searchBar]}
+          autocompleteColor="#FFFFFF"
```

Comment:
This still won't pass through, as it's not an accepted prop of SearchBar (and shouldn't be!). (I think) the props you need to pass to SearchBar are:
```jsx
<SearchBar
  ...
  color="white"
  placeholderTextColor="white"
  ...
/>
```
Now looking at it, seems a bit over the top making a new constant in global styles for `WHITE: #FFFFFF` when `"white"` is already a CSS constant. If were were going for some sort of off white throughout the app I'd see the see the use case.

I think it'd be good to review how reactjs, props and JSX work together. [This egghead course](https://egghead.io/courses/start-learning-react) helped me a lot a few years ago. In particular, [Write More Reusable React Components with Composable APIs](https://egghead.io/lessons/react-write-more-reusable-react-components-with-composable-apis) is a relevant lesson. I encourage you to watch the whole course though, it's quick and a fairly comprehensive base of react and JSX. The [official docs](https://reactjs.org/docs/components-and-props.html) are of high quality also.

Reply:
Actually, just the prop `color` may be needed if you read the `SearchBar` props.

Reply:
@Chris-Petty You are right. autocompleteColor was not a valid prop name in searchBar. I tried to create a new const as I thought that's what it meant but it's not. Thanks for reviewing the code. I also add placeholderTextColor={WHITE} to searchbar as you mentioned. 
I think I need to learn more and use it more to remember how it works as it's easily forgotten even more after long break or a holiday.